ui/src/app/components/routing-spinner/component.ts

```diff
@@ -24,7 +24,7 @@ import {routeDataStore} from 'app/utils/stores';
   styleUrls: ['./component.css'],
 })
 export class RoutingSpinnerComponent implements OnInit {
-  @ViewChild('pageSpinner') pageSpinner: ElementRef;
+  @ViewChild('pageSpinner', {static: true}) pageSpinner: ElementRef;
```

Comment:
new param required in Angular 8.  Not sure what value to choose here, but tests failed when these were false.  I have not checked them individually.